proto/model/tx.proto

```diff
@@ -55,6 +57,8 @@ message MsgUpdateModel {
   string user_manual_url = 10 [(gogoproto.moretags) = "validate:\"omitempty,url,startsnotwith=http:,max=256\""];
   string support_url = 11 [(gogoproto.moretags) = "validate:\"omitempty,url,startsnotwith=http:,max=256\""];
   string product_url = 12 [(gogoproto.moretags) = "validate:\"omitempty,url,startsnotwith=http:,max=256\""];
+  string lsf_url = 13 [(gogoproto.moretags) = "validate:\"required_unless=LsfRevision 0,omitempty,url,startsnotwith=http:,max=256\""];
+  int32 lsf_revision = 14 [(gogoproto.moretags) = "validate:\"required_with=LsfUrl,gte=0,lte=65535\""];  
```

Comment:
According to https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/service_device_management/DistributedComplianceLedger.adoc#1512-lsfrevision,
> this value SHALL monotonically increase by 1. 

Should we add this check to DCL (that a new_value=old_value+1 for the same lsfUrl)? BTW `handler_test` can be used for a unit test here.

Reply:
Should we allow user to set it, or just make it auto increase whenever there is a LsfURL change.

Reply:
As I understand, `lsf_revision` can be changed for the same `lsf_url`, right? This means that we should allow setting a new `lsf_revision` w/o setting/changing  `lsf_url`. DCL must check that new_lsf_revision = old_lsf_revision + 1.

In general, it looks like we have the following cases for editing LDF fields:
1) `lsf_url` is not changed ("" passed), and `lsf_revision` is changed. DCL must check that 
   - lsf_url already set for that model (non-empty strong)
   - new_lsf_revision = old_lsf_revision + 1.
2) `lsf_url` is changed (non-empty string which is different from the previous value is passed). `lsf_revision` is kept the same (0 passed). If there were no lsf for that model before (so that model's lsf_url is "" and lsf_revision is 0), then `lsf_revision` must be set to 1 by DCL.
3) Both `lsf_url` and `lsf_revision` are changed. DCL must check that new_lsf_revision = old_lsf_revision + 1.

Reply:
OK.. will update with this logic.